q2_types/feature_data/tests/test_transformer.py

```diff
@@ -223,6 +224,55 @@ def test_biom_v210_format_no_md_to_tsv_taxonomy_format(self):
                 BIOMV210Format, TSVTaxonomyFormat,
                 filename=os.path.join('taxonomy', 'feature-table_v210.biom'))
 
+    def test_taxonomy_format_to_metadata(self):
+        _, obs = self.transform_format(TaxonomyFormat, qiime2.Metadata,
+                                       os.path.join('taxonomy',
+                                                    '3-column.tsv'))
+
+        index = pd.Index(['seq1', 'seq2'], name='Feature ID', dtype=object)
+        exp = pd.DataFrame([['k__Foo; p__Bar', '-1.0'],
+                            ['k__Foo; p__Baz', '-42.0']], index=index,
+                           columns=['Taxon', 'Confidence'], dtype=object)
+
+        self.assertEqual(set(index), obs.ids())
```

Comment:
Good call --- the `qiime2.Metadata.__eq__/__ne__` PR wasn't merged when I wrote up this PR!